tools/bazel.rc

```diff
@@ -38,3 +38,6 @@ build:clang-msan --copt -fsanitize=memory
 build:clang-msan --linkopt -fsanitize=memory
 build:clang-msan --define tcmalloc=disabled
 build:clang-msan --copt -fsanitize-memory-track-origins=2
+
+# Test options
+test --test_env=HEAPCHECK --test_env=PPROF_PATH
```

Comment:
I think this is preferable.  I'll do some testing.  I think PPROF_PATH defaults to pprof (so it should find it if you have it installed).  I'm not sure what happens if pprof doesn't exist.  HEAPCHECK should work okay without using pprof anyway.  It apparently just makes the heap check better.